subsys/bluetooth/mesh/rpr_srv.c

```diff
@@ -267,13 +267,15 @@ static void scan_ext_report_send(void)
 	bt_mesh_model_msg_init(&buf, RPR_OP_EXTENDED_SCAN_REPORT);
 	net_buf_simple_add_u8(&buf, BT_MESH_RPR_SUCCESS);
 	net_buf_simple_add_mem(&buf, srv.scan.dev->uuid, 16);
-	if (!(srv.scan.dev->flags & BT_MESH_RPR_UNPROV_FOUND)) {
+
+	if (srv.scan.dev->flags & BT_MESH_RPR_UNPROV_FOUND) {
+		net_buf_simple_add_le16(&buf, srv.scan.dev->oob);
```

Comment:
Just double checking. Is this about the bolded sentence (section 4.4.5.3):
> When the Remote Provisioning Extended Scan procedure completes without receiving an advertisement from the unprovisioned device, the OOBInformation and AdvStructures fields shall be skipped. **When the obtained data is empty, the AdvStructures field shall be skipped.** The Status field shall be set to Success.
?

Reply:
Yes that is correct.

Reply:
Would be great to add this to the commit message for better clarity.